students/ZhuojingXie/lesson04/file_lab.py

```diff
@@ -0,0 +1,19 @@
+'''Write a program which prints the full path for all files in
+the current directory, one per line'''
+
+import os
+import pathlib
+
+cwd = os.getcwd()
+pth = pathlib.Path(cwd)
+for f in pth.iterdir():
+    print(f)
+
+
+'''Write a program which copies a file from a source,
+to a destination (without using shutil, or the OS copy command). '''
+
+with open('my_photo.jpg','rb') as rf:
+    with open('a_new_copy.jpg','wb') as wf:
```

Comment:
Note that you can open two files in one line with the with statement:

`with open('my_photo.jpg','rb') as rf, open('a_new_copy.jpg','wb') as wf:`